spec/models/mechanic_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe Mechanic, type: :model do
+  describe "relationships" do
+    it { should have_many(:rides).through(:mechanic_rides) }
+  end
```

Comment:
There were quite a few associations that got added to the `Mechanic` model that went untested.